src/object.d

```diff
@@ -3057,10 +3058,47 @@ void* aaLiteral(Key, Value)(Key[] keys, Value[] values) @trusted pure
 
 alias AssociativeArray(Key, Value) = Value[Key];
 
+/***********************************
+ * Allocate memory for an empty associative array.
+ * This is useful to avoid copying a null reference when each reference
+ * should point to the same associative array data.
+ * Without using this function, an associative array is initialized only
+ * when the first element is added.
+ * Params:
+ *      aa = The associative array.
+ * See_Also: $(LREF clear)
+ */
+void initialize(T : Value[Key], Value, Key)(ref T aa) @trusted
```

Comment:
I don't like filling up the `object` module with a lot of symbols. This is a pretty common name as well.